components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -1563,6 +1563,19 @@ impl Animatable for InnerMatrix2D {
     }
 }
 
+impl ComputeSquaredDistance for InnerMatrix2D {
+    #[inline]
+    fn compute_squared_distance(&self, other: &Self) -> Result<SquaredDistance, ()> {
+        // If possible, we should convert the InnerMatrix2D into types with physical meaning.
```

Comment:
Is that something that should be done? If yes, add "FIXME: " before the comment.